### DISCUSSION TITLE
Is Coqui no longer maintained?

### DISCUSSION THREAD

**Discussion by tgwasira** [coqui-ai/TTS#4048] [General]
Hello. Thank you for this project!

I see on coqui.ai it says "Coqui is shutting down." Does this mean that this project is no longer maintained?

Thanks again.

**Comment by eginhard** [comment]
This repository, no. But we're actively maintaining a fork: https://github.com/idiap/coqui-ai-TTS

**Reply by tgwasira** [reply]
Great, thanks.
